<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialOptDialog</class>
 <widget class="QWidget" name="SerialOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="3">
    <widget class="QComboBox" name="cBPort">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>COM1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Label3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="3">
    <widget class="QComboBox" name="cBBitRate">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="Label1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Bitrate (bps)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="Label2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Byte Size</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cBByteSize">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>7 bits</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 bits</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="Label4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Parity</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="Label5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Stop Bits</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="Label8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Flow Control</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QComboBox" name="cBFlowControl">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RTS/CTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XON/XOFF</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="cBStopBits">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>1 bit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 bits</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="cBParity">
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QSpinBox" name="cBTcpPort"/>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QCheckBox" name="cBOutputTcpPort">
     <property name="text">
      <string>Output Received Stream to  TCP Port :</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cBPort</tabstop>
  <tabstop>cBParity</tabstop>
  <tabstop>cBBitRate</tabstop>
  <tabstop>cBStopBits</tabstop>
  <tabstop>cBByteSize</tabstop>
  <tabstop>cBFlowControl</tabstop>
  <tabstop>cBOutputTcpPort</tabstop>
  <tabstop>cBTcpPort</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
